### DISCUSSION TITLE
RDS CT only shows the time, not the date.

### DISCUSSION THREAD

**Discussion by andimik** [PE5PVB/TEF6686_ESP32#105] [Ideas]
Despite of TMC (which makes no sense without the location table in my opinion, if someone really needs it, please use https://github.com/windytan/redsea ) there is still one important RDS information missing:

Some stations use the actual time, but do not set the right date. So it will be useful to show the CT date as well.

IEC 62106 says, 

> In RDS, Clock Time includes the date.

> 5.2 Clock Time and date (CT)
Time and date codes use Coordinated Universal Time (UTC) and Modified Julian Day (MJD). 
Details of using these codes, which are intended to update a free running clock in a receiver, 
are given in 6.5 and Annex B. If MJD = 0, the receiver shall not update the day/date. The 
listener, however, will not use this information directly and the conversion to local time and 
date will be made in the receiver's circuitry. CT is also used as time stamp by various RDS 
applications and thus it shall be accurate.

**Comment by PE5PVB** [comment]
There is no room left on the screen to show the date.

**Comment by andimik** [comment]
What about showing time and date alternating each second? When you use two digits for the year, we just need three characters more to show.

The longest PTY text is "Serious Classical", but I don't know if this is the broadest with that font ...

But I don't know if this might be annoying ...

```
21:44
01/11/23
21:44
01/11/23
```
